app/store/services/helpers/gps.js

```diff
@@ -16,42 +16,42 @@ const computeDistance = (
   return R * c / 1000;
 };
 
-const isInArea = (
-  position = { longitude: 0, latitude: 0 },
-  point = { longitude: 0, latitude: 0 },
+const isInRegion = (
+  currentLocation = { longitude: 0, latitude: 0 },
+  reference = { longitude: 0, latitude: 0 },
   Δλ = 0,
   Δφ = 0
 ) =>
-  Math.abs(position.longitude - point.longitude) < Δλ &&
-  Math.abs(position.latitude - point.latitude) < Δφ;
+  Math.abs(currentLocation.longitude - reference.longitude) < Δλ &&
+  Math.abs(currentLocation.latitude - reference.latitude) < Δφ;
 
-const computePathDistance = (points = []) =>
-  points.reduce((distance, currentPoint, index) => {
-    const nextPoint = points[index + 1];
+const computePathDistance = (locations = []) =>
+  locations.reduce((distance, currentPoint, index) => {
+    const nextPoint = locations[index + 1];
     if (nextPoint) {
       return distance + computeDistance(currentPoint, nextPoint);
     }
     return distance;
   }, 0);
 
-const findClosestPoint = (points = [], currentPosition = {}) => {
-  const distances = points.map((currentPoint, index) => ({
+const findClosestPoint = (locations = [], currentLocation = {}) => {
```

Comment:
i believe findClosestEdge to be more usefull

Reply:
dunno!